samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs

```diff
@@ -49,8 +49,10 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                 await GetSingleMemberAsync(turnContext, cancellationToken);
             else if (text.Contains("update"))
                 await CardActivityAsync(turnContext, true, cancellationToken);
+            else if (text.Contains("aadid"))
+                await MessageAllMembersAsync(turnContext, cancellationToken, true);
             else if (text.Contains("message"))
-                await MessageAllMembersAsync(turnContext, cancellationToken);
+                await MessageAllMembersAsync(turnContext, cancellationToken, false);
```

Comment:
What is the false boolean for?

Reply:
When the value is false, instead of sending a proactive message through AADId, the code will follow the existing function of sending proactive messages using member data.